libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -846,6 +851,31 @@ - (void)biometricAuthenticationFlowDidComplete:(NSNotification *)notification {
     return [SFScreenLockManagerInternal shared];
 }
 
+#pragma mark - Native Login
+
+- (id <SFNativeLoginManager>)useNativeLoginWithConsumerKey:(nonnull NSString *)consumerKey
+                                               callbackUrl:(nonnull NSString *)callbackUrl
+                                              communityUrl:(nonnull NSString *)communityUrl
+                                 nativeLoginViewController:(nonnull UIViewController *)nativeLoginViewController
+                                                     scene:(nullable UIScene *)scene {
+    
+    [SFSDKAppFeatureMarkers registerAppFeature:kSFAppFeatureNativeLogin];
+    NSString *key = scene ? scene.session.persistentIdentifier : kSFDefaultNativeLoginViewControllerKey;
```

Comment:
if there's no scene you could also default to `[[SFSDKWindowManager sharedManager] defaultScene]` and use its identifier so you don't have to track a separate key

Reply:
Oh, nice.  Thank you.

Reply:
As it turns out this isn't viable since `defaultScene` is nil at this point so I can't use it as a key. 😞

Reply:
ooo, where in the app delegate / scene delegate is it called from?

Reply:
I have been calling it from AppDelegate init (for non-multi-window apps).  If it's called from SceneDelegate it isn't a problem.  

However, if I called it in `didFinishLaunchingWithOptions` it would work so I guess if we document it that is fine.  There is no way that `defaultScene` will change throughout a run of the app though, correct?

Reply:
not if it's a single window app, but no preference about changing it! I was just curious about where the nil case was coming from